src/esportsbot/db_gateway_v1.py

```diff
@@ -0,0 +1,99 @@
+from sqlalchemy import create_engine, inspect
+from sqlalchemy.orm import sessionmaker
+from esportsbot.models import *
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+db_string = f"postgresql://{os.getenv('PG_USER')}:{os.getenv('PG_PWD')}@{os.getenv('PG_HOST')}:5432/{os.getenv('PG_DATABASE')}"
+
+db = create_engine(db_string)
+
+Session = sessionmaker(db)
+session = Session()
+
+base.metadata.create_all(db)
+
+print("[DATABASE] - Models created")
+
+
+class DBGatewayActions():
+    """
+    Base class for handling database queries
+    """
+    def __init__(self):
+        super().__init__()
```

Comment:
You're calling `super()` but the class doesn't have any inheritance - am I missing something?